spec/process-manager-spec.js

```diff
@@ -0,0 +1,53 @@
+/** @babel */
+
+import fs from 'fs-plus'
+import temp from 'temp'
+import path from 'path'
+import ProcessManager from '../lib/process-manager'
+
+describe('ProcessManager', () => {
+  let processManager, tempPath
+
+  function constructCommand (fileName) {
+    const filePath = path.join(tempPath, fileName)
```

Comment:
Let's pass in `tempPath` as an argument to the function, and move the function outside of the `describe` block?

It doesn't seem likely to be a problem anytime soon, but if tests were suddenly executed in parallel, we would potentially be affected by a race condition here by relying on the captured `tempPath` to not be changed by another thread/process.